1_html-css/index.html

```diff
@@ -0,0 +1,146 @@
+        interaction. What drives me is the perfect blend of aesthetics and
+        functionality that makes websites shine. This excitement to create
+        visually appealing and functional websites is what propels me to dive
+        deeper into the realm of front-end development. The more I learn, the
+        more motivated I am to refine my skills and create amazing online
+        experiences.
+      </p>
+    </section>
+
+    <section id="projects">
+      <h2>My Projects</h2>
+      <section>
+        <h3>FreeCampCode</h3>
+        <ul>
+          <li>
+            <a
+              href="https://cool-baklava-6f12d3.netlify.app"
+              target="_blank"
+              rel="noopener noreferrer"
+              >FreeCodeCamp Survey Form Project</a
+            >
+          </li>
+          <li>
+            <a
+              href="https://velvety-cajeta-66b4f4.netlify.app"
+              target="_blank"
+              rel="noopener noreferrer"
+              >FreeCodeCamp Tribute Page</a
+            >
+          </li>
+        </ul>
+      </section>
+      <section>
+        <h3>HachYourFuture</h3>
+        <p>A responsive website built using HTML, CSS.</p>
+        <a
+          href="https://calm-beignet-11ba2f.netlify.app"
+          target="_blank"
+          rel="noopener noreferrer"
+          >Live Demo</a
+        >
+        <div class="github-link">
+          <a
+            href="https://github.com/Sevda-NZ"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <i class="fab fa-github">Visit my Github</i>
+          </a>
+        </div>
+      </section>
+    </section>
+
+    <section id="skills">
+      <h2>Skills</h2>
+      <ul class="flex skill">
+        <li><i class="fab fa-html5"></i> HTML</li>
+        <li><i class="fab fa-css3"></i> CSS</li>
+        <li><i class="fab fa-js"></i> Basic JavaScript</li>
+      </ul>
+    </section>
+
+    <section id="contact">
+      <h2>Contact Me</h2>
+      <p>
+        You can reach me at sevdanaziri@email.com
+        <span class="gmail-logo"> <i class="fas fa-envelope"></i></span>
+      </p>
+      <div class="social-icons">
+        <a
+          href="https://www.facebook.com/profile.php?id=100004987278952"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <i class="fab fa-facebook"></i>
+        </a>
+        <a
+          href="https://www.instagram.com/sevdanazirii/"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <i class="fab fa-instagram"></i>
+        </a>
+        <a
+          href="https://www.linkedin.com/in/sevda-naziri-562099123/"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <i class="fab fa-linkedin"></i>
+        </a>
+      </div>
+    </section>
+
+    <footer>
+      <p>&copy; 2023 Sevda Naziri</p>
+    </footer>
+  </body>
+</html>
```

Comment:
Really good job with the portfolio. Good use of the semantic HTML elements and CSS animations.